photon-server/build.gradle

```diff
@@ -155,3 +156,31 @@ jacocoTestReport {
         }))
     }
 }
+
+
+remotes {
+    rpi {
+        host = 'photonvision.local'
```

Comment:
Is there a way we can make this user-settable? i.e. Gloworm will have a hostname of `gloworm.local`, and editing the build.gradle to adjust for that isn't ideal.

Reply:
Understood and makes sense, I hadn't considered that.

I think I could just add gloworm as another host and have it search for whichever one it can find on the network. Would make support transparent to the developer.

If we do want the user to pass in the info, pi and gloworm would be the same, but other targets might need other config (username/password/directories/etc.) I think this might get cumbersome to do fully on the command line. 

FWIW I had added this just to help myself - got tired of running the same commands over and over again during testing :). I'd be fine pulling it back out and punting on it for now. I picked gradle fairly arbitrarily, a shell script would work just fine too for deployment.

Reply:
06f7dfde1a03ad9030a21f93bd50f5914c4eeef6 should have a new implementation that should search through a list of hosts (which I added `gloworm.local` to). 

Longer-ish term, if we know other targets we'd want to support a deploy to, we can add configuration as needed.

Again, totally ok if this is a can of worms we don't want to open now, it's easy to take out.

Reply:
49b3d26ccf90bf12a02edbe51519089135314927 should also add the ability to specify -PtgtIP="X.X.X.X" to skip host searching and just use the specified info (while assuming `pi` settings for the rest of everything)